src/components/Home/HomeHeader.tsx

```diff
@@ -1,28 +1,34 @@
-import Link from 'next/link';
 import styled from 'styled-components';
 import Title from '@/components/Title.styled';
-import { useRouter } from 'next/router';
-
-const navLinks = [
-  {
-    path: '/',
-    name: '내 링크',
-  },
-  { path: '/mark', name: '내 마크' },
-];
+import { useState } from 'react';
+import { RootState, useAppDispatch } from '@/store';
+import { homeSlice } from '@/store/slices/homeSlice';
+import { useSelector } from 'react-redux';
 
 const HomeHeader = () => {
-  const router = useRouter();
-  const { pathname } = router;
+  const { name } = useSelector((state: RootState) => state.home);
+  const [isLink, setIsLink] = useState<boolean>(name === '내 링크');
```

Comment:
```suggestion
  const [isLink, setIsLink] = useState<boolean>(name === '내 링크'); << 이 부분
```
초기값을 name==='내 링크' 로 설정해서 페이지 이탈 시 다시 돌아올 때 값을 유지할 수 있어요 굳